### DISCUSSION TITLE
I CAN'T LOAD MY ROUTES

### DISCUSSION THREAD

**Discussion by vblacken** [flybywiresim/aircraft#3417] [Help] [answered]
I CAN'T LOAD MY FLIGHT PATHS BECAUSE IT DOESN'T DETECT THE POINTS AND DIVES 

![image](https://user-images.githubusercontent.com/78318935/107228850-e2efb480-69fb-11eb-93cc-65b45c8c26c9.png)

**Comment by robinottawa** [comment, marked as answer]
Sorry but I don't follow. If you get no answers, maybe clarify a little? What does "load flight paths" mean with respect to detecting waypoints.

**Reply by vblacken** [reply]
I'm sorry I use the translator, which I'm saying is, today morning I wanted to make a flight and the points of my route didn't load and then I pass that q you see in the photo

**Reply by robinottawa** [reply]
There were no route points in your flight plan? You could put them in manually. 

But if you're asking if anyone else had this problem, no, I have not. I use the MSFS world planner and the route is always in the flight computer when I start. I use the custom/experimental version.

**Comment by tuffrep** [comment]
I have had a similar problem. I use SimBrief to create a flight plan and load it directly to the MCDU but the route seems to be scaled down to a few waypoints and not the number of waypoints suggested by SimBrief. This has only just started to happen a couple of days ago. I'm not sure whether this a sim issue or an issue with the latest development mod (I have updated to the latest mod but the sim hasn't been updated for a while)

**Reply by ExampleWasTaken** [reply]
This is a known issue and currently under  investigation.
